experimental/skills/DialogToDialog/DialogRootBot/Extensions/TurnContextEx.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
+
+namespace Microsoft.BotBuilderSamples.DialogRootBot.Extensions
+{
+    public static class TurnContextEx
```

Comment:
I think this is already a part of the SDK: [ITurnContextExtensions.TraceActivityAsync](https://github.com/microsoft/botbuilder-dotnet/blob/4.7/libraries/Microsoft.Bot.Builder/ITurnContextExtensions.cs).

Reply:
Replaced by the one in the SDK. thanks

---
In reply to: [356236236](https://github.com/microsoft/BotBuilder-Samples/pull/2032#discussion_r356236236) [](ancestors = 356236236)